Src/Program.cs

```diff
@@ -148,7 +148,15 @@ private void RunInstall(string extensionPath)
                 var installableExtension = _extensionManager.CreateInstallableExtension(extensionPath);
                 var identifier = installableExtension.Header.Identifier;
                 UninstallSilent(identifier);
-                _extensionManager.Install(installableExtension, perMachine: false);
+
+                var perMachine = false;
+                if(installableExtension.Header.AllUsers != perMachine)
+                {
+                    Console.Write(string.Format("Changing `AllUsers` to {0} ... ", perMachine));
+                    SetAllUsers(installableExtension, perMachine);
+                }
+                SetAllUsers(installableExtension, perMachine);
```

Comment:
Maybe I'm missing something subtle but it looks like this is the exact line that is inside the `if` statement above.